README.md

```diff
@@ -85,38 +67,50 @@ classDef thin_slice fill:#CCFFCC,stroke:#333,stroke-width:4px
 
 The pipeline uses artificial HES data, which was chosen as it is "like" real data used in our industry, but also freely available. 
 
-> Describe the pipeline and why we've made it this way
+This example pipeline uses Apache Spark, which will be installed locally in your environment when you go through the "Getting Started" steps below.
 
-- Get the artificial HES data
-        - This can be a specific code or demographic
+The pipeline follows three steps which are common to almost all analytical pipelines:
 
+1. Getting the data - in this case we download the artificial HES data as a CSV which is saved into folder called 'data_in' on your machine (see the code in src/data_ingestion)
+2. Processing the data - the data is aggregated using Spark's python API, PySpark (the code for this is in src/processing)
+3. Saving the processed data - the processed data is saved as a csv in a folder called 'data_out' (see the code in src/data_exports)
 
 ## Prerequisites
 
-> If applicable, list the items a user needs to be able to use your repository, such as a certain version of a programming language. It can be useful to link to documentation on how to install these items.
-
-* Python (> 3.0)
+This code requires Python (> 3.0), the official Python website has [instructions for downloading and installing python](https://wiki.python.org/moin/BeginnersGuide/Download).
 
 ## Getting Started
 
-> Tell the user how to get started (using a numbered list can be helpful). List one action per step with example code if possible.
-
 1. Clone the repository. To learn about what this means, and how to use Git, see the [Git guide](https://nhsdigital.github.io/rap-community-of-practice/training_resources/git/using-git-collaboratively/).
 
 ```
-git clone <insert URL>
+git clone https://github.com/NHSDigital/RAP_example_pipeline_python
 ```
 
 2. Set up your environment, _either_ using [pip](https://pypi.org/project/pip/) or [conda](https://www.anaconda.com/). For more information on how to use virtual environments and why they are important,. see the [virtual environments guide](https://nhsdigital.github.io/rap-community-of-practice/training_resources/python/virtual-environments/why-use-virtual-environments/).
 
 ### Using pip
+
+If you're using Windows, enter the following commands into the Command Line or Powershell:
+
 ```
 python -m venv .venv
 .\.venv\Scripts\Activate.ps1
 python -m pip install -r requirements.txt
 ```
+
+If you're using Linux or MacOS, enter the following commands into the Terminal:
+
+```
+python -m venv .venv
+source venv/bin/activatee
```

Comment:
I think this is a typo! https://nhsdigital.github.io/rap-community-of-practice/training_resources/python/virtual-environments/venv/
```suggestion
source venv/bin/activate
```